### PR TITLE
Openssl

### LINKED ISSUE #6: Migrate to Openssl ruby standard library
The previous crypt gem was yanked as noted: https://github.com/titanous/yubikey/pull/5

### PR DESCRIPTION
**Pull request by ChrisLundquist**
Will resolve https://github.com/titanous/yubikey/issues/6

### CODE REVIEW

**Comment by ukd1** [comment]
@titanous have you got any pointers to what @ChrisLundquist and I have been doing wrong with ECB?

**Comment by ukd1** [comment]
@ChrisLundquist @titanous any ideas?

**Comment by ChrisLundquist** [comment]
Nope. @galternberg took a look at it as well and got stuck the same place. @dlundquist said he might take a look at it. It really shouldn't be this hard. I feel like there is some minor detail of the spec we don't know.

**Comment by ukd1** [comment]
Yep me too.

**Comment by maartenvg** [comment]
@ChrisLundquist @ukd1 ( and @titanous ) It really was a minor detail: padding should be disabled in this case. 

I posted a pull request to Chris with the fix so we/you can continue finishing the OpenSSL issue

**Comment by ukd1** [comment]
@maartenvg awesome!

@titanous -- will you merge?

File: lib/yubikey/otp.rb [hunk 1]
```diff
@@ -15,34 +15,43 @@ class Yubikey::OTP
...
-    @token = Crypt::Rijndael.new(@aes_key, 128).decrypt_block(@token)
-    
+
+    decrypter = OpenSSL::Cipher.new('AES-128-ECB').decrypt
+    decrypter.key = @aes_key
+    decrypter.padding = 0
+
+    @token = decrypter.update(@token) + decrypter.final
+
+#    @token = Crypt::Rijndael.new(@aes_key, 128).decrypt_block(@token)
```
Comment (titanous) [inline lib/yubikey/otp.rb:45]: This line should be removed.

**Pull request merged**